android/app/src/main/java/com/pillarproject/wallet/PillarFirebaseMessagingService.java

```diff
@@ -0,0 +1,62 @@
+package com.pillarproject.wallet;
+
+import android.content.Intent;
+import android.support.v4.content.LocalBroadcastManager;
+import android.util.Log;
+
+import com.facebook.react.HeadlessJsTaskService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import java.util.Map;
+
+import io.intercom.android.sdk.push.IntercomPushClient;
+import io.invertase.firebase.Utils;
+import io.invertase.firebase.messaging.RNFirebaseBackgroundMessagingService;
+import io.invertase.firebase.messaging.RNFirebaseMessagingService;
+
+public class PillarFirebaseMessagingService extends RNFirebaseMessagingService {
+    private static final String TAG = "PillarFirebaseMessaging";
+    private final IntercomPushClient intercomPushClient = new IntercomPushClient();
+    
+    @Override public void onNewToken(String refreshedToken) {
```

Comment:
This method doesn't take any action as `RNFirebaseMessagingService` doesn't contain method `onNewToken`, do we really need it? @tkulpa